qt_binder/tests/test_raw_widgets.py

```diff
@@ -70,3 +63,20 @@ def test_lookup_widget(self):
     def test_lookup_grid_layout(self):
         self.assertIs(binder_registry.lookup_by_type(QtGui.QGridLayout),
                       BasicGridLayout)
+
+
+class TestGroupBox(unittest.TestCase):
+
+    def test_group_box_can_be_childless(self):
+        box = GroupBox()
+        box.construct()
+        box.configure()
+        self.assertIsInstance(box.qobj, QtGui.QGroupBox)
+
+    def test_groupbox_alignment_works(self):
+        # PySide has a bug. Ensure that we work around it.
+        box = GroupBox()
+        box.construct()
+        box.configure()
+        _ = box.alignment
```

Comment:
The error actually occurs during the `.get()`. `.set()` does happen to call `.get()` in order to get the `old` value to pass to the trait change handler, but there is no guarantee that it must go through the same code path; that's just the current implementation detail. So I test both independently.